drivers/eeprom/Kconfig.rv3028

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 ANITRA system s.r.o.
+# SPDX-License-Identifier: Apache-2.0
+
+config EEPROM_RV3028
+	bool "Micro Crystal RV3028 User EEPROM support"
+	default y
+	depends on RTC_RV3028
```

Comment:
You should also add this dependency
```
        depends on DT_HAS_MICROCRYSTAL_RV3028_ENABLED
```
the parent needs to have a driver, and have status "okay"

Reply:
Hmm, I thought it is already covered by RTC_RV3028 config, which depends on it

Reply:
right, that is true, disregard my comment :)